transactions.go

```diff
@@ -34,14 +34,27 @@ type Transaction struct {
 	Voidable         NullBool
 	Refundable       NullBool
 	IPAddress        net.IP
-	CVVResult        CVVResult // Read only
-	AVSResult        AVSResult // Read only
-	AVSResultStreet  string    // Read only
-	AVSResultPostal  string    // Read only
-	CreatedAt        NullTime  // Read only
+	TransactionError TransactionError // Read only
```

Comment:
Let's set this to `*TransactionError` type so the user knows if it's nil or not.